File: Java/vko3/Osoite.java
public class Osoite {

    private String mKatu;
    private String mPostinumero;
    private String mKunta;

    public Osoite (String aKatu, String aPostinumero, String aKunta) {
        mKatu = aKatu;
        mPostinumero = aPostinumero;
        mKunta = aKunta;
    }

    public void setKatuosoite(String aKatu) {
        mKatu = aKatu;
    }

    public void setPostinumero(String aPostinumero) {
        mPostinumero = aPostinumero;
    }

    public void setKunta(String aKunta) {
        mKunta = aKunta;
    }

    public String getKatuosoite() {
        return mKatu;
    }
    public String getPostinumero() {
        return mPostinumero;
    }
    public String getKunta() {
        return mKunta;
    }
    
    public void tulostaTiedot() {
        System.out.println("Osoitetiedot:");
        System.out.println("Katuosoite: " + getKatuosoite());
        System.out.println("Postinumero ja toimipaikka: " + getPostinumero() + " " + getKunta());
    }
}

File: C++/Vko7/osoite.cpp
#include "osoite.h"
#include <iostream>
#include <string>

using std::cout;
using std::endl;
using std::string;

osoite::osoite() {

}

osoite::osoite(const string& aKatu, const string& aKunta)
: mKatu(aKatu), mKunta(aKunta) {   
}

osoite::~osoite() {

}

void osoite::setKatu(const string &aKatu) {
    mKatu = aKatu;
}

void osoite::setKunta(const string &aKunta) {
    mKunta = aKunta;
}

void osoite::tulostaOsoite() const {
    cout << "Osoite: " << mKatu << ", " << mKunta << endl;

}